.github/workflows/release.yml

```
@@ -17,44 +17,37 @@ jobs:
     name: Build package
     runs-on: ubuntu-latest
     container:
-      image: ubuntu:16.04
+      image: ubuntu:18.04
     steps:
       - name: Install dependencies
         run: |
           apt update -y
           apt install --no-install-recommends -y \
-            curl ca-certificates make gcc g++ clang-4.0 \
-            pkg-config git bzip2 zstd
-
-      - name: Install rust toolchain
+            ca-certificates curl git \
+            clang make gcc g++
+      - name: Install Rust toolchain
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ env.RUST_TOOLCHAIN }}
-
       - name: Fetch latest code
         uses: actions/checkout@v3
-
       - name: Build
-        run: cargo build --release --target x86_64-unknown-linux-gnu
-
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --locked --target x86_64-unknown-linux-gnu
       - name: Collect shared
         run: |
-          mkdir -p shared/crab-parachain-runtime
-          mkdir -p shared/darwinia-parachain-runtime
-          tar cjSf shared/darwinia-collator-x86_64-linux-gnu.tar.bz2 \
-            -C target/x86_64-unknown-linux-gnu/release \
+          mkdir -p shared
+          tar cjSf \
+            shared/darwinia-collator-x86_64-linux-gnu.tar.bz2 \
+            -C \
+            target/x86_64-unknown-linux-gnu/release \
             darwinia-collator
-          tar cf shared/darwinia-collator-x86_64-linux-gnu.tar.zst \
-            -C target/x86_64-unknown-linux-gnu/release \
-            darwinia-collator -I pzstd
-
-          mv target/x86_64-unknown-linux-gnu/release/wbuild/crab-parachain-runtime/*.wasm shared/crab-parachain-runtime
-          mv target/x86_64-unknown-linux-gnu/release/wbuild/darwinia-parachain-runtime/*.wasm shared/darwinia-parachain-runtime
-
       - name: Upload shared
         uses: actions/upload-artifact@v2
         with:
-          name: darwinia-artifact
+          name: bridger-artifact
```

```suggestion
          name: darwinia-artifact
```

darwinia-artifact is better than bridger in this repository and the download shared is used name darwinia-artifact